packages/extension-vscode/src/server.ts

```diff
@@ -157,10 +164,14 @@ const loadWebHint = async (directory: string): Promise<hint.Engine> => {
 
         return engine;
     } catch (e) {
-        // Instantiating webhint failed. Prompt the user to retry after checking their configuration.
+        // Instantiating webhint failed, log the error to the webhint output panel to aid debugging.
+        console.log(e);
+
+        // Prompt the user to retry after checking their configuration.
         const retry = 'Retry';
         const answer = await connection.window.showErrorMessage(
-            'Unable to start webhint. Check your `.hintrc` and ensure dependencies are installed.',
+            'Unable to start webhint. Ensure you are using `hint 3.4.3` or newer and dependencies are installed. ' +
```

Comment:
> `3.4.3`

Can this not be hard-coded? Things change quickly sometimes.

Reply:
@alrra how about "Ensure you are using the latest version of `hint`"?